stone-payments/emerald-components-android
# Button Customizations

#### What this PR does?
Modifies emerald button attributes fix #51 

#### How should it be manually tested?
1.  Open button example acitivity

2. Check if the following is customizable:
- [x] Background color
- [x] Text color
- [x] Border color
- [x] Button text

3. Check if the following behaviors occurs
- [x] Outline: Text color and border color should be the same
- [x] Darken the button by 20% when the user clicks
